tests/board_test.py

```diff
@@ -0,0 +1,20 @@
+from ..lib.board import *
+
+
+class TestBoard:
+    def test_move(self):
+        #     move method changes the value of the given key( number from the user) from empty to the given mark
+        new_board = Board()
+        new_board.move(1, "X")
+        assert new_board.board[1] == "X"
+
+    def test_position_taken(self):
+        # position_taken method will check if the value of the given key is not
+        # not emply and return True
+        new_board = Board()
+        new_board.move(1, "X")
+        assert new_board.position_taken(1) == True
+
+    def test_board_full(self):
```

Comment:
You should be testing both possible values for boolean methods. So you should have a test that returns true when the board is full. Please update the MR with this change.

Reply:
I'm working on it write now.Thank you!

Reply:
That is what this method is doing, though, no? The way I read it is exactly how you defined `board_is_full`. What is the difference between `board_full` and `board_is_full`? When would `board_full` not return false?